go/tools/bazel/bazel_test.go

```diff
@@ -70,7 +70,7 @@ func createPaths(paths []string) error {
 }
 
 func TestRunfile(t *testing.T) {
-	file := "go/tools/bazel/README.md"
+	file := "README.md"
```

Comment:
I think if this is changed to a file in the test's working directory, it won't cover the case where `Runfile` needs to look in the runfiles directory, which is what we really want to test.

I'm not sure whether `Runfile` itself will need to change in order for tests to use it when run from the local workspace and from "remote", named workspaces.